tello_sim/ursina_adapter.py

```
@@ -23,11 +27,57 @@
+# Default gate layout: alternating heights (>= 30 cm clearance) and varied
+# diameters (23-50 cm) along the flight corridor. Overridden by gates.json if present.
+DEFAULT_GATES = [
+    {'color': 'yellow', 'x': -15, 'z': 45,  'diameter_cm': 30, 'clearance_cm': 30, 'yaw': 0},
+    {'color': 'green',  'x': -15, 'z': 100, 'diameter_cm': 45, 'clearance_cm': 55, 'yaw': 0},
+    {'color': 'red',    'x': -14, 'z': 200, 'diameter_cm': 23, 'clearance_cm': 40, 'yaw': 0},
+    {'color': 'blue',   'x': -15, 'z': 270, 'diameter_cm': 50, 'clearance_cm': 70, 'yaw': 0},
+    {'color': 'violet', 'x': -15, 'z': 340, 'diameter_cm': 35, 'clearance_cm': 45, 'yaw': 0},
+]
+
+
+def _clamp(value, low, high):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_clamp`

(ANN202)